data/json/recipes/recipe_ammo.json

```diff
@@ -299,77 +268,195 @@
   },
   {
     "type": "recipe",
-    "result": "arrow_wood_heavy",
+    "result": "bolt_wood",
     "category": "CC_AMMO",
     "subcategory": "CSC_AMMO_ARROWS",
     "skill_used": "fabrication",
-    "skills_required": [ [ "archery", 5 ], [ "survival", 1 ] ],
+    "skills_required": [ [ "archery", 4 ], [ "survival", 1 ] ],
+    "using": [ [ "forging_standard", 1 ] ],
     "difficulty": 5,
-    "time": 18000,
+    "time": 30000,
     "autolearn": true,
     "book_learn": [ [ "recipe_arrows", 3 ] ],
-    "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "CUT", "level": 1 } ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "CUT", "level": 1 }, { "id": "ANVIL", "level": 3 } ],
+    "tools": [ [ [ "tongs", -1 ] ] ],
     "components": [
       [ [ "adhesive", 1, "LIST" ], [ "filament", 20, "LIST" ] ],
-      [ [ "steel_chunk", 1 ], [ "scrap", 3 ] ],
-      [ [ "stick", 2 ], [ "broom", 2 ], [ "2x4", 2 ], [ "bee_sting", 2 ] ],
+      [ [ "steel_tiny", 1, "LIST" ] ],
+      [ [ "stick", 1 ], [ "broom", 1 ], [ "2x4", 1 ], [ "bee_sting", 1 ] ],
       [ [ "fletching", 1, "LIST" ] ]
     ]
   },
   {
     "type": "recipe",
-    "result": "arrow_metal",
```

Comment:
Is it just me or is there no longer a arrow_metal recipe at all?  I have had issues seeing the 'aluminium broadhead' arrows in the game - not available even via debug.

Reply:
It was intentionally removed until we have the capability to make an aluminum extrusion rig.

Reply:
Would it not be less surprising (at least for existing characters) to leave the metal arrow in until the recipe can be updated with new requirements?  Just wondering.

Reply:
Removing the recipe doesn't remove resulting item.

Reply:
I realise this, my point is that the recipe that used to be there is not there any longer until some future time - meanwhile the capability that used to be there disappeared without any reason.  Does this make the game more fun?

Reply:
1) It makes very little difference.
2) Argue with Kevin if you care that much.

Reply:
Just scrap removed recipes into mod if you care.

Reply:
Oh? Can you tell me what mod it is? (Purely out of curiosity)

Reply:
The one you will create :)

Reply:
Ah. I misunderstood.

Reply:
How does one go about creating a mod?  Is there a document somewhere?  I'm quite new to the game.

Reply:
Start here - https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-contributors.